bot/constants.py

```diff
@@ -285,6 +285,7 @@ class Roles(NamedTuple):
     owner = 267627879762755584
     helpers = int(environ.get("ROLE_HELPERS", 267630620367257601))
     core_developers = 587606783669829632
+    everyone = 267624335836053506
```

Comment:
this should probably copy from https://github.com/python-discord/sir-lancebot/blob/522bea4fe05e74dc3d6a02f97585221a9da7cd2c/bot/constants.py#L133

Reply:
I agree.